recipes/r-boot/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = 'boot' %}
+{% set version = '1.3-18' %}
+{% set sha256 = '12fd237f810a69cc8d0a51a67c57eaf9506bf0341c764f8ab7c1feb73722235e' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-{{ name|lower }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: {{ name }}_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/{{ name }}_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/{{ name }}/{{ name }}_{{ version }}.tar.gz
+
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  # skip on windows because of this error:
+  # https://ci.appveyor.com/project/conda-forge/staged-recipes/build/1.0.10696/job/hma4wc1ms5ssj2qk#L415
+  skip: true  # [win or linux32]
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+
+requirements:
+  build:
+    - r-base
```

Comment:
Weirdly, during conversion, it seems to think `r-base` doesn't exist for `osx-64`. Not entirely sure why it would think that. Particularly as `r-base` does exist for `osx-64` and on `conda-forge` no less. This breaks conversion particularly for this recipe. Any ideas?